.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env", "@babel/preset-react"]
```

Comment:
Is there a a need for babel here still?

Reply:
Nope good call! All the webpack/babel stuff you caught is old, I will push a commit removing these and then merge